pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -516,6 +517,19 @@ def handle_command(
     if args.pkgname == "scipy":
         scipy_fixes(new_args)
 
+    # FIXME: For some unknown reason,
+    #        opencv-python tries to link a same library (libopencv_world.a) multiple times,
+    #        which leads to 'duplicated symbols' error.
+    if args.pkgname == "opencv-python":
+        duplicated_lib = "libopencv_world.a"
```

Comment:
At the final link time, it tries to link the same library multiple times. I couldn't find a way to prevent this from happening.